loader/src/main/kotlin/net/weavemc/loader/mixin/MixinSandbox.kt

```diff
@@ -57,10 +57,10 @@ class SandboxedMixinLoader(
 ) : ClassLoader(parent) {
     private val injectedResources = mutableMapOf<String, ByteArray>()
     private val tempFiles = mutableMapOf<String, URL>()
-    private val loaderExclusions = hashSetOf("net.weavemc.loader.mixin.MixinAccess")
+    private val loaderExclusions = mutableSetOf("net.weavemc.loader.mixin.MixinAccess")
```

Comment:
This actually creates a linked hash set, which is ever so slightly slower than a regular hash set. Not an issue, but just interesting.

Reply:
I have an IDE ruleset for strict kotlin abstractions like those, and since it wasn't performance-heavy I didn't think much of it.